doc/manual/src/installation/installing-binary.md

```diff
@@ -131,8 +131,6 @@ and `/etc/zshrc` which you may remove.
    ```console
    sudo launchctl unload /Library/LaunchDaemons/org.nixos.nix-daemon.plist
    sudo rm /Library/LaunchDaemons/org.nixos.nix-daemon.plist
-   sudo launchctl unload /Library/LaunchDaemons/org.nixos.activate-system.plist
-   sudo rm /Library/LaunchDaemons/org.nixos.activate-system.plist
```

Comment:
Erg. Sorry. I think my previous comment didn't connect the dots.  I assume it was supposed to be the nix-store daemon instead of the nix-Darwin one?

Reply:
Oh, is that the one that mounts the volume? What's that called again, don't have access to a machine with the newer setup currently?

Reply:
Yep! I don't actually either, but it is: `/Library/LaunchDaemons/org.nixos.darwin-store.plist`